frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/import/import_panel.dart

```diff
@@ -66,38 +67,7 @@ class ImportPanel extends StatelessWidget {
   Widget build(BuildContext context) {
     final width = MediaQuery.of(context).size.width * 0.7;
     final height = width * 0.5;
-    return FlowyContainer(
-      Theme.of(context).colorScheme.surface,
-      height: height,
-      width: width,
-      child: GridView.count(
-        childAspectRatio: 1 / .2,
-        crossAxisCount: 2,
-        children: ImportType.values
-            .where((element) => element.enableOnRelease)
-            .map(
-              (e) => Card(
-                child: FlowyButton(
-                  leftIcon: e.icon(context),
-                  leftIconSize: const Size.square(20),
-                  text: FlowyText.medium(
-                    e.toString(),
-                    fontSize: 15,
-                    overflow: TextOverflow.ellipsis,
-                    color: Theme.of(context).colorScheme.tertiary,
-                  ),
-                  onTap: () async {
-                    await _importFile(parentViewId, e);
-                    if (context.mounted) {
-                      FlowyOverlay.pop(context);
-                    }
-                  },
-                ),
-              ),
-            )
-            .toList(),
-      ),
-    );
+    return _buildKeyboardListeningFlowyContainer(context, height, width);
```

Comment:
There's no reason to wrap the whole returning widget in a method. Just return the widget.